prac_05/hex_colours.py

```diff
@@ -0,0 +1,13 @@
+
```

Comment:
make it so that I don't need to have case sensitive names. for example, skyblue should be able to work

Reply:
Good point. Thanks! 
Sorry for such a late code review request.